src/mWater_collate/grab_sensor_notes.R

```diff
@@ -0,0 +1,47 @@
+
+grab_mWater_sensor_notes <- function(){
+
+  #source clean mwater script for all notes cleaned
+
+  source("src/mWater_collate/clean_mwater_notes.R")
```

Comment:
to do: If you source this at the beginning of the targets script I don't think you should have to source it here. If you can you should check if that is the case.

Reply:
Checked and you are correct.